src/search.cpp

```diff
@@ -223,8 +223,8 @@ template uint64_t Search::perft<true>(Position&, Depth);
 
 
 /// MainThread::search() is called by the main thread when the program receives
-/// the UCI 'go' command. It searches from root position and at the end prints
-/// the "bestmove" to output.
+/// the UCI 'go' command. It searches from the root position and at the end
+/// prints the "bestmove" to output.
```

Comment:
The expression "to print to output" is rather clumsy, because "to output" is already a verb.
Simply "to output", instead of "to print to ouput".